openssl-sys/src/handwritten/aes.rs

```diff
@@ -37,3 +28,17 @@ extern "C" {
         inlen: c_uint,
     ) -> c_int;
 }
+cfg_if! {
```

Comment:
This doesn't need a cfg_if, it can just be left where it was previously and tagged with the cfg directly.

Reply:
fixed.

Reply:
fixed it .